internal/internal_task_handlers.go

```diff
@@ -301,23 +309,40 @@ func isCommandEvent(eventType enumspb.EventType) bool {
 	}
 }
 
-// NextCommandEvents returns events that there processed as new by the next command.
-// TODO(maxim): Refactor to return a struct instead of multiple parameters
-func (eh *history) NextCommandEvents() (result []*historypb.HistoryEvent, markers []*historypb.HistoryEvent, binaryChecksum string, sdkFlags []sdkFlag, msgs []*protocolpb.Message, err error) {
+// NextTask returns the next task to be processed.
+func (eh *history) NextTask() (*preparedTask, error) {
```

Comment:
```suggestion
func (eh *history) nextTask() (*preparedTask, error) {
```

Can we not expose this? May need to change the name if there's a clash. Just helps me confirm that we don't expose via something someone might use via some interface somewhere. Not required though.

Reply:
done